docs/how-to-guides/s3_configuration.md

```diff
@@ -0,0 +1,17 @@
+To configure an S3 bucket you'll have to set the following configuration options with the appropriate values for your existing bucket `juju config [charm_name] [configuration]=[value]`:
```

Comment:
Does this suffice?

Reply:
I think it's really about allowing you to scale the application to multiple units (because otherwise you have files being uploaded to one of the units only.

Reply:
If you scale up without the bucket the files will be present in each one of the containers anyway. Perhaps I'm missing something?

Reply:
Fixed